File: 백준/Bronze/31789. 모험의 시작/모험의 시작.c
#include <stdio.h>

int main(void){
  int N, X, S;
  scanf("%d", &N);
  scanf("%d %d", &X, &S);
  int dap = 0;
  for (int i = 0; i <= N; i++){
    int c, p;
    scanf("%d %d", &c, &p);
    if (c <= X && dap < p){
      dap = p;
    }
  }
  if (dap > S){
    printf("YES");
  }
  else {
    printf("NO");
  }
  return 0;
}

File: 백준/Bronze/31789. 모험의 시작/모험의 시작.java
import java.util.Scanner;


public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int X = sc.nextInt();
        int S = sc.nextInt();
        int dap = 0;
        for (int i = 1; i<=N; i++){
            int c = sc.nextInt();
            int p = sc.nextInt();
            if (c <= X && dap < p){
                dap = p;
            }
        }
        if (dap > S){
            System.out.println("YES");
        }
        else {
            System.out.println("NO");
        }
    }
}